sc/src/test/java/uk/ac/ebi/atlas/search/JsonBioentityInformationControllerWIT.java

```diff
@@ -12,9 +12,7 @@
 import org.springframework.test.web.servlet.setup.MockMvcBuilders;
 import org.springframework.web.context.WebApplicationContext;
 
-import static org.hamcrest.Matchers.greaterThanOrEqualTo;
-import static org.hamcrest.Matchers.hasSize;
-import static org.hamcrest.Matchers.isA;
+import static org.hamcrest.Matchers.*;
```

Comment:
It’s good practice to just bring the imports you need. Intent is clearer and it’s impossible you end up having conflicts.